chemprop/features/utils.py

```diff
@@ -101,12 +101,13 @@ def load_valid_atom_features(path: str, smiles: List[str]) -> List[np.ndarray]:
         features_df = features_df.applymap(lambda x: np.array(x.replace('\r', '').replace('\n', '').split(',')).astype(float))
 
         # Truncate by number of atoms
-        num_atoms = {x: Chem.MolFromSmiles(x).GetNumAtoms() for x in features_df.index.to_list()}
+        # FV and EH, probably to get rid of H atoms, but not the right way to do it!
+        # num_atoms = {x: Chem.MolFromSmiles(x).GetNumAtoms() for x in features_df.index.to_list()}
```

Comment:
Since we took out this part, we should remove `from rdkit import Chem` from the top of the file, since it is used nowhere else (and thus triggers an "empty import"). I think we can also remove the commented out part. Since we use this for atom and bond feature read in, it doesn't make sense anymore to keep it (or am I missing something?)

Reply:
I removed it :)